pkg/util/trace/buffer_pipe.go

```diff
@@ -398,6 +410,7 @@ func (t batchCSVHandler) NewItemBuffer(name string) bp.ItemBuffer[bp.HasName, an
 	case MOStatementType:
 		opts = append(opts, bufferWithFilterItemFunc(filterTraceInsertSql))
 	case MOErrorType:
+	case MOStatsType:
 	default:
 		panic(fmt.Errorf("unknown type %s", name))
```

Comment:
moerr, I know this is old code.   But, since you have edited the line above, lets get rid of this one.